ui/core/src/model/variables.ts

```diff
@@ -13,29 +13,54 @@
 
 import { Definition, JsonObject } from './definitions';
 
-/**
- * Variable definition options that are common to all variables.
- */
-export interface VariableDefinition<Options extends JsonObject = JsonObject> extends Definition<Options> {
-  display: VariableDisplayOptions;
-  selection: VariableSelectionOptions;
-  capturing_regexp?: string;
+export const DEFAULT_ALL_VALUE = '$__all' as const;
+
+export type VariableName = string;
```

Comment:
Yeah exactly, I'm not sure the exact way of doing it. But I have seen some projects do with types like Duration. But it might be too much and a string is simple enough